test/ui-test/testSuites/suite_status/tst_wallet/test.feature

```diff
@@ -43,4 +43,13 @@ Feature: Status Desktop Wallet
 
     	Examples:
       	  | account_name | seed_phrase |
-      	  | one		     | indoor dish desk flag debris potato excuse depart ticket judge file exit |
\ No newline at end of file
+      	  | one		     | indoor dish desk flag debris potato excuse depart ticket judge file exit |
+
+ 	Scenario Outline: User delete a generated account
+        When the user generate a new account with <account_name> and TesTEr16843/!@00
```

Comment:
COMMENT / REFLEXION / QUESTION: `account_name` can be passed as a property in the `context` and then maybe it is just only needed in the `WHEN` statement. 
I'm not sure which is the best approach here, if parametrizing each single statement or expecting a param as a context . WDYT?

Reply:
Resolved in chat